webapp/tests/elster_client/test_elster_client.py

```diff
@@ -576,16 +576,27 @@ def test_set_form_data_dict_results_in_dict_with_est_data_field_and_meta_data_fi
         self.assertIn('est_data', result)
         self.assertIn('meta_data', result)
 
-    def test_set_form_data_dict_results_in_est_data_dict_with_same_keys(self):
+    def test_set_form_data_dict_results_in_est_data_dict_with_same_keys_and_adapted_merkzeichen_keys(self):
+        _MERKZEICHEN_KEYS = {
+            'person_a_beh_grad': 'person_a_disability_degree',
+            'person_a_blind': 'person_a_has_merkzeichen_bl',
+            'person_a_gehbeh': 'person_a_has_merkzeichen_g',
+            'person_b_beh_grad': 'person_b_disability_degree',
+            'person_b_blind': 'person_b_has_merkzeichen_bl',
+            'person_b_gehbeh': 'person_b_has_merkzeichen_g'
+        }
         form_data = LotseMultiStepFlow(None)._DEBUG_DATA[1]
         with patch('app.elster_client.elster_client.current_user', MagicMock(is_active=True, is_authenticated=True)):
             result = _generate_est_request_data(form_data)
 
         for key in form_data.keys():
-            self.assertIn(key, result['est_data'])
+            if key in _MERKZEICHEN_KEYS:
+                self.assertIn(_MERKZEICHEN_KEYS[key], result['est_data'])
+            else:
+                self.assertIn(key, result['est_data'])
 
     def test_yes_str_for_bool_keys_result_in_true(self):
-        bool_strs = {}
+        bool_strs = {'person_a_blind': 'yes', 'person_b_blind': 'yes', 'person_a_gehbeh': 'yes', 'person_b_gehbeh': 'yes'}
```

Comment:
Wait..how does this work? You just deleted the fields from the BOOL_KEYS list - so why are they converted?

Reply:
The keys `person_a_blind`, `person_b_blind`, `person_a_gehbeh`and `person_b_gehbeh` are the keys from the form. They are not converted to booleans because Erica expects `person_a_has_merkzeichen_bl` and so on. 
In `_generate_est_request_data` we set `person_a_has_merkzeichen_bl`, ... on accordingly to the values of `person_a_blind`, `person_b_blind`, ... . These are then converted to booleans.

Before we could just take all keys in `_BOOL_KEYS` and set them. Now the keys we are converting are different from the input keys. That is why we set all those keys in line 599. However, we still only test that the keys in _BOOL_KEYS are converted (s. line 605).

Reply:
Okay...I was just confused by the naming of `bool_strs` then - because they're really not^^
But no problem, I don't think this is something we'd wanna have more often or for a long time, just till the fields are created, right? Then let's jsut keep it as is^^